### ISSUE TITLE
gvm led

### ISSUE THREAD

**Issue by mmabba5** [bitfocus/companion-module-requests#1170]
- [ ] **I have researched the list of existing Companion modules and requests and have determined this has not yet been requested**

The name of the device, hardware, or software you would like to control:
https://gvmled.com/
https://gvmled.com/download-gvm-app/

What you would like to be able to make it do from Companion:
what the app does but using companion like for Brightness I can either use the knob on the stream deck plus or a button for raising and one  button for lowering it, and the same thing for color temperature and rgb and a  button to select which light I’m trying to control 

Direct links or attachments to the ethernet control protocol or API:

**Comment by showem88** [comment]
I agree, I would very much like a companion module (Connection) to control GVM lights. These are the most popular LED event lighting panels and can only be controlled by a bluetooth app. It would be VERY helpful and popular to control these lights from Companion.
